documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1737,17 +1672,176 @@ Typically, you configure the {prodname} SQL Server connector in a `.json` file u
 endif::community[]
 
 ifdef::product[]
+To deploy a {prodname} SQL Server connector, you add the connector files to Kafka Connect, create a custom container to run the connector, and then add connector configuration to your container.
+For details about deploying the {prodname} SQL Server connector, see the following topics:
+
+* xref:deploying-debezium-sql-server-connectors[]
+* xref:sql-server-connector-properties[]
+endif::product[]
+
+ifdef::product[]
+// Type: procedure
+// ModuleID: deploying-debezium-sql-server-connectors
+[[sql-server-deploying-a-connector]]
+=== Deploying {prodname} SQL Server connectors
+
+To deploy a {prodname} SQL Server connector, you must build a custom Kafka Connect container image that contains the {prodname} connector archive, and then push this container image to a container registry. 
+You then need to create the following custom resources (CRs):
+
+* A `KafkaConnect` CR that configures your Kafka Connector and that specifies the name of the image that you created to run your {prodname} connector.
+  You apply this CR to the OpenShift Kafka instance.
```

Comment:
What is the "OpenShift Kafka instance"?

Reply:
@gunnarmorling Hmm. I didn't coin that phrase. I'm taking it directly from the PG connector deployment topic that these updates are based on. I take it to mean the Kafka instance that's deployed on with AMQ Streams in an OpenShift environment.  Is there a better way to refer to that?

Reply:
Perhaps just omit that sentence, I don't think it adds much value.

Reply:
I intially didn't see them as saying the same thing., but I think that sentence basically repeats, in different words, the sentence in the next bullet, "You apply this CR to the OpenShift instance where Red Hat AMQ Streams is deployed." Is that right?  

I've modified that one to read as follows:

_"You apply this CR to the OpenShift instance where link:https://access.redhat.com/products/red-hat-amq#streams[Red Hat {StreamsName}] is deployed."_   

  `//In the downstream doc, the {StreamsName} variable resolves to AMQ Streams.`

I also bumped the following sentence from the _Prerequisites_ section up to immediately follow the preceding sentence:
_"{StreamsName} offers operators and images that bring Kafka to OpenShift."_

Reply:
Sounds good. Only "that bring Apache Kafka...". Generally, I think we should use "Apache Kafka" at the first occurrence on a page for sure, and also in such "landmark" sentence as the one you describe. Just "Kafka" is fine in follow-ups and explanations of details. It's not a black-or-white rule, I hope you can make something out of it.

Reply:
Yeah, a new habit to get into. Lots of instances of the terminology sprinkled throughout the doc. I'll be more mindful of applying the correct form!